data/Sri Guru Granth Sahib Ji/0187.json

```diff
@@ -1141,7 +1141,7 @@
         "translations": {
           "English": {
             "Dr. Sant Singh Khalsa": {
-              "translation": "O my mind, seek the One,",
+              "translation": "O my mind, seek the One",
```

Comment:
Perhaps this and the next line should be split as follows:

```
ry mn myry qUM qw kau Awih ]
O my mind,

jw kY aUxw kChU nwih ]1]
seek the One who lacks nothing. ||1||
```

Reply:
The current method is correct, `ਜਾ ਕੈ ਊਣਾ ਕਛਹੂ ਨਾਹਿ` translates to `who lacks nothing`.

Reply:
The suggestion does not move the phrase "who lacks nothing". So please re-read the suggestion.

Reply:
`ਤੂੰ ਤਾ ਕਉ ਆਹਿ` means to `seek the One`